keywording/text.xml

```diff
@@ -434,6 +434,36 @@ keyword, so packages are not to be marked stable for one of these architectures.
 </body>
 </subsection>
 
+<subsection>
+<title>Keeping track of pending stabilizations</title>
+<body>
+
+<p>
+Maintainers need some method or system to organize and start pending
+stabilizations once they become due.
+</p>
+
+<p>
+There are several options available:
+</p>
+
+<ul>
+  <li>
+    Use <c>imlate</c> from app-portage/gentoolkit
+  </li>
+  <li>
+    Use packages.gentoo.org's maintainer pages (which have a Stabilization
+    tab)
+  </li>
+  <li>
+    Use <c>pkgcheck</c>'s <c>StableRequest</c> check, e.g.
+      <c>grep -ri "larry@" */*/metadata.xml -l | cut -d'/' -f1-2 | xargs pkgcheck scan -k StableRequest</c>
```

Comment:
I'd love to add a script to [mgorny/mgorny-dev-scripts](https://github.com/mgorny/mgorny-dev-scripts) that just does this.

Reply:
Me too. It's _almost_ there with the components but not quite.